lib/spack/spack/cmd/common/arguments.py

```diff
@@ -111,17 +143,13 @@ def _specs(self, **kwargs):
     '-L', '--very-long', action='store_true',
     help='show full dependency hashes as well as versions')
 
-_arguments['jobs'] = Args(
-    '-j', '--jobs', action='store', type=int, dest='jobs',
-    help="explicitely set number of make jobs. default is #cpus")
-
 _arguments['tags'] = Args(
     '-t', '--tags', action='append',
     help='filter a package query by tags')
 
 _arguments['jobs'] = Args(
-    '-j', '--jobs', action='store', type=int, dest="jobs",
-    help="explicitly set number of make jobs, default is #cpus.")
+    '-j', '--jobs', action=SetParallelJobs, type=int, dest='jobs',
+    help='explicitly set number of parallel jobs')
```

Comment:
probably this could say "parallel build jobs"

Reply:
I left that generic because we might want to use `-j` in cases where we don't actually build things, but we spawn multiple processes anyhow - like running tests or analyzing log files  in parallel. Would that make sense?

Reply:
sure -- makes sense